pkg/virt-controller/services/renderresources.go

```diff
@@ -199,7 +199,7 @@ func WithAutoMemoryLimits(namespace string, namespaceStore cache.Store) Resource
 	}
 }
 
-func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
+func WithCPUPinning(cpu *v1.CPU, cpuManagerPolicyBetaOptionsAnnotationValue string) ResourceRendererOption {
```

Comment:
Please consider changing the type here to the enum, and adding an `Unknown` or `Unset` value:
```suggestion
func WithCPUPinning(cpu *v1.CPU, cpuManagerPolicyBetaOptions v1.CPUManagerPolicyBetaOptions) ResourceRendererOption {
```

Reply:
The advantage is that it limits the number of possible inputs to the string constants comprising the `CPUManagerPolicyBetaOptions` type (and not any string value).

Indeed, some function higher in the call stack will have to perform this conversion.

Reply:
DONE